lock-api/src/main/java/com/palantir/lock/client/metrics/TimeLockFeedbackBackgroundTask.java

```diff
@@ -96,11 +96,11 @@ private void scheduleWithFixedDelay() {
                                 .serviceName(serviceName)
                                 .namespace(namespace)
                                 .build();
+                        Optional<LeaderElectionStatistics> maybeStatistics =
```

Comment:
Is this the bug that you found? If so, nice.

Reply:
Yeah, the method is not idempotent, so two nodes would get virtually no impact data